homework_1/task1/fizzbuzz/fizzBuzz.go

```diff
@@ -0,0 +1,40 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func FizzBuzz(start, end int) ([]string, error){
+	if start > end {
+		return nil, errors.New("range start is greater than range end")
+	}
+
+	if start <= 0 {
+		return nil, errors.New("start is 0 or negative")
+	}
+
+	if end <= 0 {
+		return nil, errors.New("end is 0 or negative")
+	}
+
+	var changedOutput []string
+
+	for i := start; i <= end; i++ {
+		switch{
```

Comment:
Should be a whitespace before `{` always except when creating a new struct.
You can automatically format your code with `go fmt ./...` from `homework_2` directory and it should format all .go files in that directory and its child directories.

Reply:
Neat trick, thank you! I was doing it by hand 😄